src/typescript/azure-openapi-validator/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Ignore autogenerated files
+
+rules/index.ts
```

Comment:
Why do we need this again?

Reply:
This avoids checking in the auto-generated rules indexing file produced by `gulp generate/typescript`.